Emus/PICO/Pico8 Wrapper - Splore.sh

```diff
@@ -20,9 +20,8 @@ else
 fi
 
 main() {
-	#echo 1008000 > /sys/devices/system/cpu/cpu0/cpufreq/scaling_min_freq
```

Comment:
Removed this comment as the script should be clean and free of commented code. A clean code practice.